model/_types/query_dsl/abstractions.smithy

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: Apache-2.0
+// 
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
```

Comment:
Is it appropriate to name the file as Abstractions?

Reply:
Yes, as we are abstracting data we can name file as *Abstractions*.